### PR TITLE
get all messages

### PR DESCRIPTION
**Pull request by TheDivineAurora**
Issue: #108

### CODE REVIEW

**Comment by TheDivineAurora** [comment]
please tell if any additions needed

File: src/controllers/message.controller.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+exports.getAllMessages = async (req, res) => {
+  try {
+    const SubChannelId = req.body.subchannel.id;
+    const isValidSubchannel = await SubChannel.findById(SubChannelId).populate('messages');
+
+    if (!isValidSubchannel) {
+      return response_400(res,'Missing required fields');
+    }
+
+    const messages = isValidSubchannel.messages;
```
Comment (automatefreely) [inline src/controllers/message.controller.js]: You need the mark the messages some way. To display the receive message in left and send message  to right.

File: src/controllers/message.controller.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+} = require("../utils/responseCodes.utils");
+
+
+
+exports.getAllMessages = async (req, res) => {
+  try {
+    const SubChannelId = req.body.subchannel.id;
+    const isValidSubchannel = await SubChannel.findById(SubChannelId).populate('messages');
+
+    if (!isValidSubchannel) {
```
Comment (automatefreely) [inline src/controllers/message.controller.js]: this is not required as this check is already in middleware

**Review by automatefreely — COMMENTED** [review]
Resolve these issues

**Pull request closed without merging**